UM-USElab/gradecraft-development
# Copy badge from badge index

### Status
**READY**

### Description
This PR makes it possible to copy badges as an instructor from the badge index page.

### Todos

### Migrations
NO

### Steps to Test or Reproduce

1. Navigate to the badge index. 
2. Click the Gear drop-down menu
3. Click "Copy Badge"
4. Confirm that the badge copied is the same as the previous one (files, unlocks, images) but has 'Copy of…" prepended to the title 


### Impacted Areas in Application
List general components of the application that this PR will affect:

* 

======================
Closes #[ISSUE NUMBER]
